script/cadence/scripts/get-key-index.cdc

```diff
@@ -0,0 +1,5 @@
+pub fun main(addr: Address, keyIndex: Int): Int {
```

Comment:
Do we have a way to list all keys of a given address?

Reply:
We can loop over the keys, e.g. to get the total number of keys:
```cadence
pub fun main(addr: Address, keyIndex: Int): Int {
    let acct = getAccount(addr)
    var index = 0
    while true {
        let key = acct.keys.get(keyIndex: index)
        if let num = key {
            index = index + 1
        } else {
            return index
        }
    }
    return 0
}
```